### ISSUE TITLE
Language setting not working properly (with web content?)

### ISSUE THREAD

**Issue by gitgetgot** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#2654]
Hi,
[screenshot](https://picload.org/image/rwggrdii/screenclip.png)
Same on Mac and Windows versions.

I've set up "English" as Choosen Language in Desktop Settings, because I want everything shown in  english language inside the fine Google-Play-Music-Desktop-Player.

Unfortuanately, it seems (orange on the screenshot) the webcontent will still shown in German language. Only items of the Player itself (red on the screenshot) seems to be in english language.

So that language setting does not really make sense (at least for me) :-)

Can you do anything about that?

**Comment by MarshallOfSound** [comment]
@gitgetgot This is by design, the "Desktop Settings" window is settings just for the desktop app itself, if you want to change the language GPM is displayed in you have to change your google account's language as that is the language that GPM displays it's content in

**Comment by Landen13** [comment]
@MarshallOfSound 
How can auto-translation of artist names be disabled in player, if the Web interface works properly? In the Web interface auto-translation doesn't work (I have set English as a primary language), but inside player artist names are translated into Russian. How this can be fixed?

I disabled Google settings to watch my locations, hope this will help, but not yet.

Auto-translated artists are scrobbled into Last.fm incorrectly, and statistic is broken. This is the main problem of auto-translation.
